sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -274,11 +274,11 @@ var detectionCondition = new MetricWholeSeriesDetectionCondition()
 
 var detectionConfiguration = new AnomalyDetectionConfiguration(metricId, configurationName, detectionCondition);
 
-Response<AnomalyDetectionConfiguration> response = await adminClient.CreateMetricAnomalyDetectionConfigurationAsync(detectionConfiguration);
+Response<string> response = await adminClient.CreateMetricAnomalyDetectionConfigurationAsync(detectionConfiguration);
```

Comment:
Did we also consider making this a model type we control that contains the Id? Just in case there is ever more information we'd like to return in the future, it wouldn't need to be a breaking change.

Reply:
We have an issue for this: https://github.com/Azure/azure-sdk-for-net/issues/16325
Discussed with Java about this possibility. We'll ask the service about this and consider how these methods are intended to grow before making this change. I'm personally in favor of adding doing it as well.